Makefile

```diff
@@ -219,7 +219,7 @@ clidocsgen: ensure-gopath
 
 
 .PHONY: codegen-local
-codegen-local: ensure-gopath mod-vendor-local notification-docs notification-catalog gogen protogen clientgen openapigen clidocsgen manifests-local
+codegen-local: gogen protogen clientgen openapigen manifests-local
```

Comment:
revert?

Reply:
Must have been accidental. I've fixed this.